server/svix-server/src/v1/endpoints/message.rs

```diff
@@ -60,23 +60,28 @@ pub struct MessageIn {
     pub uid: Option<MessageUid>,
     #[validate]
     pub event_type: EventTypeName,
-    pub payload: serde_json::Value,
+    pub payload: Option<serde_json::Value>,
```

Comment:
Making the payload optional in MessageIn makes no sense. It's not optional, it has to exist.